### ISSUE TITLE
 Temporary Denied Page Bypass not working

### ISSUE THREAD

**Issue by ivanjong** [marcelloc/Unofficial-pfSense-packages#57]
does anyone got this to work? i cant seem to get it to bypass using my old cgi php page.

**Comment by HadiJadallah** [comment]
same here.... cant get it to work

**Comment by HadiJadallah** [comment]
Dears,

Fixed.
/usr/local/pkg/e2guardianfx.conf.template is old.
I found the following parameters missing:
bypassversion = '1|2'
cgibypass = 'on|off'

I added those manually to the config file and template and it worked for me.

@Marcello:
Seems the template files used in 5.3.3 are old (perhaps from v5.0 period)

BR
Morlac

**Comment by marcelloc** [comment]
Thanks for the feedback, I'll take a look soon.

This options need to be changed while selection repot to cgi or normal html page, right?

**Comment by HadiJadallah** [comment]
No. These are important to activate the Bypass for blocked pages. Bottom of blocked page there will be an acknowledg link to allow 5minutes bypass of the block.
